ui/ace/build.sh

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (C) 2007-2020 Crafter Software Corporation. All Rights Reserved.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as published by
+# the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+rm -rf ../../static-assets/libs/ace
+cp -r node_modules/ace-builds/src-min-noconflict ../../static-assets/libs/ace
+cat src/ace-append.js >> ../../static-assets/libs/ace/ace.js
```

Comment:
@sumerjabri @jdrossl please review:

This is the bash script content that the [gradle build](https://github.com/craftercms/studio/blob/1433f32c5fc326c6e63bd7463a8f3dd6ec5aeb40/pom.xml#L655) executes.